test/ansible/PROJECT

```diff
@@ -0,0 +1,7 @@
+domain: com
+repo: wut
+resources:
+- group: cache.example.com
```

Comment:
It is more than just the group. I think it will be `cache.example` and the domain would be `com`